pkg/planner/core/physical_plans.go

```diff
@@ -736,7 +755,7 @@ type PhysicalIndexScan struct {
 	// The index scan may be on a partition.
 	physicalTableID int64
 
-	GenExprs map[model.TableItemID]expression.Expression
+	genExprs map[model.TableItemID]expression.Expression
```

Comment:
why make it for lower case？

in a later pkg move, the physical operator will be moved out of the core, and elements inside should exported for usage.